### PR TITLE
Merge RC back into master

### PR DESCRIPTION
**Pull request by thomasp85**
v3.3.0 is on CRAN 🎉

This PR merges the release branch back into master

### CODE REVIEW

File: R/geom-ribbon.r [hunk 1]
```diff
@@ -22,8 +22,8 @@
 #' @inheritParams layer
 #' @inheritParams geom_bar
 #' @param outline.type Type of the outline of the area; `"both"` draws both the
-#'   upper and lower lines, `"upper"` draws the upper lines only. `"legacy"`
-#'   draws a closed polygon around the area.
+#'   upper and lower lines, `"upper"`/`"lower"` draws the either lines only.
```
Comment (clauswilke) [inline R/geom-ribbon.r]: Something seems to have gone wrong here, "draws the either lines only" doesn't make sense. Maybe "draws the respective lines only"?

File: R/stat-boxplot.r [hunk 2]
```diff
@@ -59,7 +59,9 @@ StatBoxplot <- ggproto("StatBoxplot", Stat,
   },
 
   setup_params = function(data, params) {
-    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE, group_has_equal = TRUE)
+    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE,
+                                          group_has_equal = TRUE,
+                                          main_is_optional = TRUE)
```
Comment (clauswilke) [inline R/stat-boxplot.r]: Tidyverse style indenting?

File: R/geom-ribbon.r (hunk 1 above)
Comment (clauswilke) [inline R/geom-ribbon.r]: Something seems to have gone wrong here: "draws the either lines only". Maybe write "draws the respective lines only"?

File: R/stat-boxplot.r (hunk 2 above)
Comment (thomasp85) [reply R/stat-boxplot.r]: Isn't this in line with the style guide?

File: R/geom-ribbon.r (hunk 1 above)
Comment (thomasp85) [reply R/geom-ribbon.r]:
```suggestion
#'   upper and lower lines, `"upper"`/`"lower"` draws the respective lines only.
```

File: R/stat-boxplot.r (hunk 2 above)
Comment (yutannihilation) [reply R/stat-boxplot.r]: If we follow the tidyverse style guide, I believe these likes should be:

``` r
code <- "
    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE,
                                          group_has_equal = TRUE,
                                          main_is_optional = TRUE)
"

styler::style_text(code)
#> Warning: Could not use colored = TRUE, as the package prettycode is not
#> installed. Please install it if you want to see colored output or see `?
#> print.vertical` for more information.
#> 
#> params$flipped_aes <- has_flipped_aes(data, params,
#>   main_is_orthogonal = TRUE,
#>   group_has_equal = TRUE,
#>   main_is_optional = TRUE
#> )
```

<sup>Created on 2020-03-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

But, it seems the guide is unclear about this point (I guess the "bad" example here describes somehow related rule, but there seems no corresponding text):

https://style.tidyverse.org/syntax.html#long-lines

Comment (hadley) [reply R/stat-boxplot.r]: I'd probably write:

```R
#> params$flipped_aes <- has_flipped_aes(
#>   data, params,
#>   main_is_orthogonal = TRUE,
#>   group_has_equal = TRUE,
#>   main_is_optional = TRUE
#> )

**Pull request merged**